packages/next/src/builders/build/lib/create-package-json.ts

```diff
@@ -28,10 +56,19 @@ export async function createPackageJson(
       next: allWorkspaceDeps.next,
       react: allWorkspaceDeps.react || reactVersion,
       'react-dom': allWorkspaceDeps['react-dom'] || reactDomVersion,
+      ...projectNpmDependencies.deps,
+    },
+    devDependencies: {
+      ...projectNpmDependencies.devDeps,
     },
-    devDependencies: {},
   };
 
+  if (Object.keys(projectNpmDependencies.peerDeps).length > 0) {
+    outPackageJson.peerDependencies = {
```

Comment:
Are there any cases where the `peerDependencies` for 1 application should not be the `peerDependencies` for another application?